distribution-core/pom.xml

```diff
@@ -109,6 +109,13 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>powsybl-cgmes-import-postprocessor</artifactId>
+            <version>${project.version}</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
The introduction of a new dependency should be reported as a breaking change in the PR description.